### ISSUE TITLE
activity app does not pass occ app:check-code

### ISSUE THREAD

**Issue by jnfrmarks** [owncloud/activity#282]
### Steps to reproduce

1. Install 8.1 enterprise
2. invoke occ app:check-code

### Expected behaviour

The app passes the check

### Actual behaviour

The app fails the check:

```

[32mAnalysing /var/www/html/owncloud/apps/activity/tests/datadeleteactivitestest.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/usersettingstest.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/mailqueuehandlertest.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/mock/extension.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/apitest.php[39m
 6 errors
    [37;41mline   38: OC_User - static method call on private class[39;49m
    [37;41mline   39: OC_User - static method call on private class[39;49m
    [37;41mline   85: OC_User - static method call on private class[39;49m
    [37;41mline   89: OC_User - static method call on private class[39;49m
    [37;41mline  148: OC_User - static method call on private class[39;49m
    [37;41mline  149: OC_User - static method call on private class[39;49m
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/applicationtest.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/parameterhelpertest.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/navigationtest.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/hooksdeleteusertest.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/datatest.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/datahelpertest.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/controller/activitiestest.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/controller/settingstest.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/controller/feedtest.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/apptest.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/personaltest.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/consumertest.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/displaytest.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/grouphelpertest.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/backgroundjob/expireactivitiestest.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/backgroundjob/emailnotificationtest.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/testcase.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/tests/bootstrap.php[39m
 1 errors
    [37;41mline   16: OC_Hook - static method call on private class[39;49m
[32mAnalysing /var/www/html/owncloud/apps/activity/personal.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/templates/personal.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/templates/stream.item.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/templates/rss.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/templates/stream.body.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/templates/email.notification.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/templates/stream.app.navigation.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/templates/stream.list.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/extension/files_sharing.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/extension/files.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/controller/feed.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/controller/settings.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/controller/activities.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/lib/navigation.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/lib/parameterhelper.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/lib/consumer.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/lib/data.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/lib/api.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/lib/mailqueuehandler.php[39m
 1 errors
    [37;41mline  249: OC_Helper - static method call on private class[39;49m
[32mAnalysing /var/www/html/owncloud/apps/activity/lib/grouphelper.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/lib/hooks.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/lib/fileshooksstatic.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/lib/fileshooks.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/lib/usersettings.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/lib/backgroundjob/emailnotification.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/lib/backgroundjob/expireactivities.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/lib/datahelper.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/lib/display.php[39m
 2 errors
    [37;41mline   93: OC_Helper - static method call on private class[39;49m
    [37;41mline  105: OC_Helper - static method call on private class[39;49m
[32mAnalysing /var/www/html/owncloud/apps/activity/appinfo/app.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/appinfo/routes.php[39m
 1 errors
    [37;41mline   27: OC_API - static method call on private class[39;49m
[32mAnalysing /var/www/html/owncloud/apps/activity/appinfo/update.php[39m
 0 errors
[32mAnalysing /var/www/html/owncloud/apps/activity/appinfo/application.php[39m
 0 errors
[37;41mApp is not compliant[39;49m


```
### Server configuration

Ubuntu 14.04
PHP 5.5.9
ownCloud Enterprise Edition 8.1 alpha 1 (daily) Build:2015-04-16T03:08:13+00:00 f50798a674bf223e1c8015b9a27287b2ac6c9f1a

**Comment by nickvergessen** [comment]
Yes, but it's mostly the tests and FilesView.
FilesView will be dealt with in #249 for 8.2, the tests don't matter too much.

Will look at the other stuff soon

**Comment by nickvergessen** [comment]
Tests have been excluded now https://github.com/owncloud/core/pull/15838 so the activity app passes again

**Comment by bcutter** [comment]
Hi, when running "sudo -u www-data php /var/www/owncloud/occ app:check-code activity" I get:

"Analysing /var/www/owncloud/apps/activity/lib/display.php
 1 errors
    line   94: OC_Helper - static method call on private class
App is not compliant"

==> ???

OC v8.1.1
Activity v2.0.1

**Comment by nickvergessen** [comment]
Yeah can't really fix that without breaking previews in 8.1
You will have to live with that until 8.2 is out
